docs/integrations/integration-basics/supported-transformations.md

```diff
@@ -0,0 +1,125 @@
+---
+title: Supported Transformations
+sidebar_position: 30
+---
+
+When you create a connection to a Macrometa collection, you can add a _transformation_. These transformations are similar to Macrometa [stream workers](../../cep/), which help you work with data streams and change your data using Stream QL in stream worker queries.
```

Comment:
```suggestion
When you create a connection to a Macrometa collection, you can add a _transformation_. These transformations are similar to Macrometa [stream workers](../../cep/), which help you change your data using Stream QL in stream worker queries.
```
@diana-macrometa - Shall we not mention about the  "work with data streams"? Because the user might wonder where data streams come into play when they add a connection with a transformation.